iron-form.html

```diff
@@ -230,8 +253,13 @@
     },
 
     _registerElement: function(e) {
-      e.target._parentForm = this;
-      this._customElements.push(e.target);
+      var element = e.target;
+      element._parentForm = this;
+      this._customElements.push(element);
+
+      // Save the original value of this input.
+      this._customElementsInitialValues.push(
+          this._usesCheckedInsteadOfValue(element) ? element.checked : element.value);
```

Comment:
Perhaps use a method here that extras the value of the element, so that one does not need to write this check every time one tries to get a value from an element inside of `iron-form`?